api/v1/postgres_types.go

```diff
@@ -653,3 +664,10 @@ func enableAuditLogs(parameters map[string]string) {
 	parameters["pgaudit.log_relation"] = "on"
 	parameters["pgaudit.log_parameter"] = "on"
 }
+
+// setPostgresParams add the provided params to the parameter map
+func setPostgresParams(parameters map[string]string, providedParams map[string]string) {
+	for k, v := range providedParams {
```

Comment:
My feeling is that there should be at least be a denylist of params be configurable which a user cannot override

Reply:
In the central `cloud-api` or locally in each postgreslet/service cluster? Locally is more flexible, but also more difficult to maintain.

Reply:
unsure about this

Reply:
implementation wise, locally in a configmap is easier, deployment is done via playbook, so keeping this configmap identical between installation should also be a easy task

Reply:
I used viper to read a (apparently space-separated) list of blocked values from the environment, convert it into a map for easier use and pass that map along to the code in question. now before setting a given postgres param, we check if is contained in the blocklist. if it is, we simply ignore it.